sky/tools/gn

```diff
@@ -53,7 +53,9 @@ def to_gn_args(args):
         gn_args['target_os'] = 'ios'
         gn_args['ios_deployment_target'] = '7.0'
         gn_args['use_ios_simulator'] = args.simulator
-        if not args.simulator:
+        if args.simulator:
+            gn_args['use_libjpeg_turbo'] = False
```

Comment:
so... do jpegs work in the simulator? @chinmaygarde can you try https://github.com/flutter/flutter/issues/918 on the simulator?